master/buildbot/www/hooks/bitbucket.py

```diff
@@ -12,24 +12,161 @@
 # Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 # Copyright Buildbot Team Members
-# Copyright 2013 (c) Mamba Team
-import json
+# Copyright Manba Team
```

Comment:
I copied this from here actually: https://github.com/buildbot/buildbot/blob/master/master/buildbot/test/unit/test_www_hooks_bitbucket.py which has `# Copyright Manba Team`. Should I change that as well then?